### DISCUSSION TITLE
Shaders on hands/Held items?

### DISCUSSION THREAD

**Discussion by Spifflet** [IrisShaders/Iris#941] [Q&A] [answered]
Is there an option I'm missing somewhere or is there just no support for this yet?

**Comment by IMS212** [comment, marked as answer]
Correct, it is currently not added.

**Reply by Spifflet** [reply]
Oh ok thanks! I was beginning to worry I was missing something

**Comment by BeansBeefBroccoli** [comment]
I think this is what (this)[https://github.com/IrisShaders/Iris/pull/909] is for.
